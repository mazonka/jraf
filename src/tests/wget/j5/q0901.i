command=jw profile 1 : md 0 /.jraf.sys/users : profile 1