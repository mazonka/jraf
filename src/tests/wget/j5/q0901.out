OK a * * * * * : OK /.jraf.sys/users : AUTH